code/datums/mutations.dm

```diff
@@ -277,6 +277,23 @@
 		owner.drop_item()
 		owner.emote("cough")
 
+/datum/mutation/human/dwarfism
+
+	name = "Dwarfism"
+	quality = MINOR_NEGATIVE
+	text_gain_indication = "<span class='notice'>Everything around you seems to grow..</span>"
+	text_lose_indication = "<span class='notice'>Everything around you seems to shrink..</span>"
+
+/datum/mutation/human/dwarfism/on_acquiring(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.resize = 0.8
+	owner.visible_message("<span class='danger'>[owner] suddenly shrinks!</span>")
+
+/datum/mutation/human/dwarfism/on_losing(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.resize = 1.2
+	owner.visible_message("<span class='danger'>[owner] suddenly grows!</span>")
```

Comment:
It's probably blurry because you're using imperfect maths?

Unless I'm mistaken (I don't know the resize code), if you wanted to set their size... would it either be good to call a proc that resets size to its default?

If that doesn't exist and I'm reading this right, this does make smaller > size*0.8, make bigger > size*1.2, you'll wind up with 0.96 size (at least relative to the original size), if you want to hit 1.00, you want:
owner.resize = 1.25

But I don't know if that'll work so I'm not pretending it will.